optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -750,9 +751,9 @@ class LightGBMTuner(_LightGBMBaseTuner):
     def __init__(
         self,
         params: dict[str, Any],
-        train_set: "lgb.Dataset",
```

Comment:
Strings are specified as type because `lgb` is not imported (and thus the name `lgb` results in error) when the user doesn't have lightgbm installed. (This is controlled by `try_import()`.) Could you revert these changes?